### ISSUE TITLE
hyena equivalent to cross attention

### ISSUE THREAD

**Issue by MiqG** [HazyResearch/hyena-dna#82]
Hi, is there an example of how the Hyena block can be implemented in an equivalent manner to cross attention?

Thanks a lot in advance!

**Comment by alexeyshmelev** [comment]
@MiqG , did you solve this somehow?

**Comment by MiqG** [comment]
No... @alexeyshmelev I haven't had the time to tackle this yet, I was hoping it would be simple though
